scripts/install-worker.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script updates and installs the necessary prerequisites for an image
+# cutout backend starting with a stack container.
+
+# Bash "strict mode", to help catch problems and bugs in the shell
+# script. Every bash script you write should include this. See
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/ for details.
+# set -u is omitted because the setup bash function does not support it.
+set -eo pipefail
+
+# Enable the stack.  This should be done before set -x because it will
+# otherwise spew a bunch of nonsense no one cares about into the logs.
+source /opt/lsst/software/stack/loadLSST.bash
+setup lsst_distrib
+
+# Display each command as it's run.
+set -x
+
+# Download the image cutout backend.  This can be removed if RFC-828 is
+# accepted, since that will include the image cutout backend in lsst-distrib.
+#
+# Currently, this uses the main branch because there appears to be no
+# alternative (no releases and no tags).
```

Comment:
It'll only get tagged once it's in the stack release process for lsst_distrib; at which point you won't need this anyways.

Reply:
Jim was still talking about wanting to leave it out of lsst-distrib, in which case I think we'll want to insist that he make periodic releases, at least to the level of pushing a tag.

Reply:
Right. We'd need to have a process that tests the backend package against an `lsst_distrib` tag to ensure it still works and then tag the cutouts backend package with that `lsst_distrib` tag.